src/js/pages/jobs/JobsTaskDetail.js

```diff
@@ -0,0 +1,178 @@
+import classNames from 'classnames';
+import {Confirm} from 'reactjs-components';
+
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import ChronosStore from '../../stores/ChronosStore';
+import TaskDetail from '../services/task-details/TaskDetail';
```

Comment:
+1 to this, me and @daiyiwhale talked about this. Are you cool with doing this as a separate task?